docs/source/install/k8s_ha.rst

```diff
@@ -46,15 +46,55 @@ makes installing the complex StackStorm infrastructure as easy as:
   # Add Helm StackStorm repository
   helm repo add stackstorm https://helm.stackstorm.com/
 
-  helm install stackstorm/stackstorm-ha
+  # Update the StackStorm repository
+  helm repo update stackstorm
 
-Once the deployment is finished, it will show you the first steps to get started working with the new cluster via WebUI
-or ``st2`` CLI client:
+  # Install StackStorm HA with an automatically-generated release name in the "stackstorm" namespace
+  # Replace "--generate-name" with a release name if you would like to name the deployment
+  # Omit the "--namespace stackstorm" flag if you would like to deploy to the "default" namespace
+  # Add "--create-namespace" if the namespace that you are specifying needs to be created
+  helm install --generate-name --namespace stackstorm stackstorm/stackstorm-ha
 
-.. figure :: /_static/images/helm-chart-notes.png
-    :align: center
+After the installation completes, it will display a message similar to the following:
 
-.. todo:: Update this screenshot. It is out of date.
+.. code-block:: bash
+
+  NAME: stackstorm-ha-1657376085
+  LAST DEPLOYED: Sat Jul  9 10:14:46 2022
+  NAMESPACE: stackstorm
+  STATUS: deployed
+  REVISION: 1
+  NOTES:
+  Congratulations! You have just deployed StackStorm HA!
+
+    ███████╗████████╗██████╗     ██╗  ██╗ █████╗      ██████╗ ██╗  ██╗
+    ██╔════╝╚══██╔══╝╚════██╗    ██║  ██║██╔══██╗    ██╔═══██╗██║ ██╔╝
+    ███████╗   ██║    █████╔╝    ███████║███████║    ██║   ██║█████╔╝
+    ╚════██║   ██║   ██╔═══╝     ██╔══██║██╔══██║    ██║   ██║██╔═██╗
+    ███████║   ██║   ███████╗    ██║  ██║██║  ██║    ╚██████╔╝██║  ██╗
+    ╚══════╝   ╚═╝   ╚══════╝    ╚═╝  ╚═╝╚═╝  ╚═╝     ╚═════╝ ╚═╝  ╚═╝
```

Comment:
This looks broken in the indentation or how it's rendered.

BTW, do we really need all the Helm chart output copy-pasted into the documentation?
It might change at any time and includes the details the user might copy-paste by mistake.

The intent of the previous screenshot it just a graphical demonstration of the output.

Reply:
I prefer text to an image, but, it's your project.

Reply:
It's fine; I'll redeploy soon and upload an image ...